api/proto/teleport/legacy/types/types.proto

```diff
@@ -2416,10 +2416,10 @@ message HardwareKey {
   // By default, serial number validation is disabled.
   HardwareKeySerialNumberValidation SerialNumberValidation = 2 [(gogoproto.jsontag) = "serial_number_validation,omitempty"];
 
-  // PinCacheTimeout is the amount of time in nanoseconds that Teleport clients
+  // PinCacheTTL is the amount of time in nanoseconds that Teleport clients
   // will cache the user's PIV PIN when hardware key PIN policy is enabled.
-  int64 PinCacheTimeout = 3 [
-    (gogoproto.jsontag) = "pin_cache_timeout,omitempty",
+  int64 PinCacheTTL = 3 [
```

Comment:
@zmb3 @rosstimothy WDYT about this name change? During the RFD phase, "timeout" didn't feel right to me but "ttl" didn't come to mind until now.